src/PhpParser/Node/Manipulator/PropertyManipulator.php

```diff
@@ -31,14 +32,28 @@ final class PropertyManipulator
      */
     private $nameResolver;
 
+    /**
+     * @var ParsedNodesByType
+     */
+    private $parsedNodesByType;
+
+    /**
+     * PropertyManipulator constructor.
+     * @param BetterNodeFinder      $betterNodeFinder
+     * @param BetterStandardPrinter $betterStandardPrinter
+     * @param NameResolver          $nameResolver
+     * @param ParsedNodesByType     $parsedNodesByType
+     */
```

Comment:
These have no added value, remove it please

Reply:
I've updated the pull request